spec/features/cart/show_spec.rb

```diff
@@ -221,4 +221,47 @@
 
     expect(page).to have_content("There is no more inventory for #{@tire.name}")
   end
+
+  it "quantity of item is decreased in cart" do
+    visit "/items/#{@pull_toy.id}"
+
+    within "#item-info" do
+      click_on "Add to Cart"
+    end
+
+    visit "/items/#{@pull_toy.id}"
+
+    within "#item-info" do
+      click_on "Add to Cart"
+    end
+
+    visit "/cart"
+
+    within "#cart-item-#{@pull_toy.id}" do
+      expect(page).to have_button("-")
+      expect(page).to have_content("Quantity: 2")
+
+      click_button "-"
+    end
+
+    visit "/cart"
+
+    expect(page).to have_content("Quantity: 1")
+
+    within "#cart-item-#{@pull_toy.id}" do
+      click_button "-"
+    end
+
+    visit "/cart"
+
+    quantity_pulltoy = @cart.quantity_of(@pull_toy.id)
+    subtotal_pulltoy = @pull_toy.price * quantity_pulltoy
+
+    expect(page).to_not have_content(@pull_toy.name)
```

Comment:
Great testing here, nice work for accounting for when an item quantity is decreased to 0